src/components/DifficultySwitcher/DifficultySwitcher.tsx

```diff
@@ -23,6 +23,15 @@ const DifficultSwitcher: FC<DifficultSwitcherProps> = ({
 }) => {
   return (
     <div className={classnames("difficulty-switcher", className)}>
+      <button
```

Comment:
Now that there are 5 difficulty buttons here, I wonder if there's a way here to do this programmatically.